http-add-on/values.yaml

```diff
@@ -163,6 +163,17 @@ interceptor:
   scaledObject:
     # -- The interval (in milliseconds) that KEDA should poll the external scaler to fetch scaling metrics about the interceptor
     pollingInterval: 1
+    pendingRequestsTrigger:
+      enabled: false
+    # -- The CPU utilization threshold that KEDA should use to scale the interceptor
```

Comment:
What would happen if user don't specify a trigger here? We should not create SO then imho.